eclass/font.eclass

```diff
@@ -77,7 +79,7 @@ font_xfont_config() {
 			-e ${EPREFIX}/usr/share/fonts/encodings/large \
 			"${ED%/}/${FONTDIR}/${1//${S}/}" || eerror "failed to create fonts.dir"
 		if [[ -e fonts.alias ]]; then
-			doins fonts.alias || die "failed to install fonts.alias" # TODO old EAPI cleanup
+			doins fonts.alias
```

Comment:
could be simplified into `[[ -e fonts.alias ]] && ...` now, but just a suggestion